config/routes.rb

```diff
@@ -4,19 +4,24 @@
 
   # Sufia generated routes
   blacklight_for :catalog
-  devise_for :users
+
+  scope "/admin" do
+    devise_for :users
+  end
+
   Hydra::BatchEdit.add_routes(self)
```

Comment:
This seems weird that it doesn't have the admin prefix. Do you know what routes this creates?